atom/browser/api/atom_api_session.cc

```diff
@@ -481,6 +549,19 @@ void Session::ClearHostResolverCache(mate::Arguments* args) {
                  callback));
 }
 
+void Session::ClearAuthCache(mate::Arguments* args) {
+  ClearAuthCacheOptions options;
+  args->GetNext(&options);
```

Comment:
Should this throw an error on failure? Looks like from the initial doc the options aren't optional.